integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/ReceiptPolling.java

```diff
@@ -0,0 +1,57 @@
+package no.difi.meldingsutveksling.receipt;
+
+import no.difi.meldingsutveksling.ServiceIdentifier;
+import no.difi.meldingsutveksling.config.IntegrasjonspunktProperties;
+import no.difi.meldingsutveksling.logging.Audit;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.PostConstruct;
+import java.util.List;
+
+import static no.difi.meldingsutveksling.receipt.MessageReceiptMarker.markerFrom;
+
+/**
+ * Periodically checks non final receipts, and their respective services for updates.
+ */
+@Component
+public class ReceiptPolling {
+
+
+    private static final Logger log = LoggerFactory.getLogger(ReceiptPolling.class);
+
+    @Autowired
+    private IntegrasjonspunktProperties props;
+
+    @Autowired
+    private MessageReceiptRepository messageReceiptRepository;
+
+    @PostConstruct
+    private void addTestData() {
+        // TODO: fjernes etter test!
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1135)